app/views/subscriptions/unsubscribe.html.haml

```diff
@@ -9,5 +9,20 @@
       .govuk-panel__body
         %p= t('.confirmation')
 
-  .govuk-grid-column-two-thirds
-    %p.govuk-body=t('.resubscribe_html', link: link_to(t('.resubscribe_link_text'), new_subscription_path(search_criteria: @subscription.search_criteria_to_h), class: 'govuk-link', id: 'job-alert-link-from-cancel-unsubscribe-gtm'))
+    = form_for @unsubscribe_feedback_form, url: unsubscribe_feedback_subscription_path(@subscription) do |f|
+      = f.govuk_error_summary
+
+      %h2.govuk-heading-l
+        = t('.heading')
+
+      = f.govuk_radio_buttons_fieldset(:reason, legend: { size: 'm' }) do
```

Comment:
I think we can use the default legend size here, in which case you don't need to specify anything for legend